pkg/controller/controller_binding.go

```diff
@@ -434,7 +475,7 @@ func (c *controller) reconcileServiceInstanceCredential(binding *v1alpha1.Servic
 	// that's actually a soft delete-- i.e. we have some finalization to do.
```

Comment:
This comment is no longer accurate - the deletion timestamp may not be set, but we're doing orphan mitigation.